<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeDlg</class>
 <widget class="QDialog" name="RecipeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Recipe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/Resource/Picture/svg/tune.svg</normaloff>:/Resource/Picture/svg/tune.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>标签</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_name">
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>收集方式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_method">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Scan method.</string>
         </property>
         <item>
          <property name="text">
           <string>范围扫描</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>点位选择</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="lb_range">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>范围</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_start">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Mass begin.</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>199</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lb_arrow">
         <property name="minimumSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>→</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sb_end">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Mass end.</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="lb_points">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>点位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_points">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>225</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>225</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Mass points.</string>
         </property>
         <property name="text">
          <string>2/4/16/18/28/32/44</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>扫描间隔</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_dwell">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Point dwell time(ms).</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1204</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2048</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>每点分隔</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_ppamu">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Points per amu.</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>数据类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_unitreport">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Report unit type.</string>
         </property>
         <item>
          <property name="text">
           <string>电流</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>分压</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>气压单位</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_unitpressure">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Pressure unit for total pressure and partial pressure.</string>
         </property>
         <item>
          <property name="text">
           <string>Torr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mBar</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pa</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>激活灯丝</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_flmt">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Filament index for this recipe.</string>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>运行时长</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="te_time">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Recipe run time.</string>
         </property>
         <property name="maximumTime">
          <time>
           <hour>23</hour>
           <minute>59</minute>
           <second>59</second>
          </time>
         </property>
         <property name="minimumTime">
          <time>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
         <property name="displayFormat">
          <string>H:mm</string>
         </property>
         <property name="timeSpec">
          <enum>Qt::LocalTime</enum>
         </property>
         <property name="time">
          <time>
           <hour>1</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>倍增器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_emauto">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>EM operation method.</string>
         </property>
         <property name="text">
          <string>自动开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QTableWidget" name="tbl_recipe"/>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QComboBox" name="cb_run">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Run method.</string>
         </property>
         <item>
          <property name="text">
           <string>单次</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>连续</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_run">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>运行</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_del">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
